libmproxy/web/app.py

```diff
@@ -176,6 +177,60 @@ def post(self, flow_id):
             raise APIError(400, r)
 
 
+class ViewPluginFlowContent(RequestHandler):
```

Comment:
This should just be called `FlowContentView` or something along those lines, no? If we have something like that, we surely want to expose all existing content views! :relaxed: